src/styles.css

```diff
@@ -2,3 +2,7 @@
 
 html, body { height: 100%; }
 body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
+.icon{
```

Comment:
Change your class name .icon is global name

Reply:
Changes